### ISSUE TITLE
Default monitor profile can only be set through preferences

### ISSUE THREAD

**Issue by Thanatomanic** [Beep6581/RawTherapee#4648]
In the display profile dropdown list at the bottom of the editor window, you can select "System default (...)". However the option `autoMonitorProfile` does not get set. You can check this by going into Preferences, where the checkbox "Use operation system's main monitor colour profile" remains unchecked. I expect it to get checked.

The other way around things work: if you check the box in the preferences, the "System default (...)" option becomes the active item in the dropdown.

**Comment by agriggio** [comment]
hi, why do you consider this a bug? if you want your setting to stay permanent, you should use the preferences dialog...or maybe I am misunderstanding?

**Comment by Thanatomanic** [comment]
This issue balances on a wrong assumption of mine. Whoops. I thought it had always been like this: you choose a profile in the dropdown below the image, and it gets stored for re-use after restart. That's not the case...
I was confused because ever since I started to build RT on my own, the dropdown remained empty. I guess I had just forgotten that in my 'regular' installation, many months ago, I had ticked the checkbox in preferences.

But still: wouldn't it make sense to store the chosen profile between sessions?
